src/cbmc/cbmc_parse_options.h

```diff
@@ -50,8 +50,6 @@ class optionst;
   "(dimacs)(refine)(max-node-refinement):(refine-arrays)(refine-arithmetic)"\
   "(refine-strings)" \
   "(string-printable)" \
-  "(string-max-length):" \
-  "(string-max-input-length):" \
```

Comment:
This option is not required for CBMC at the moment as the string solver integration for C is incomplete.

Reply:
That I am aware of, but we do have `(refine-strings)` etc. So really we should not have a half-consistent view. I believe the way to go about this is as we have done with several other option sets already: define the set of options and the help strings in a header file and just use the macro.

Reply:
Fully agreed. Note, however, that `refine-strings` is default in JBMC and hence the options won't be the same for CBMC and JBMC.

Reply:
Ok, so we might need two different sets, but it would still be nice to have all this encapsulated in some header file. It's fairly easy to fix now, but will be a lot more costly later on.